xbmc/cores/VideoPlayer/VideoRenderers/windows/RendererBase.cpp

```diff
@@ -184,21 +198,13 @@ bool CRendererBase::Configure(const VideoPicture& picture, float fps, unsigned o
   m_lastHdr10 = {};
   m_HdrType = HDR_TYPE::HDR_NONE_SDR;
   m_useHLGtoPQ = false;
+  HDR_STATUS hdrStatus = CWIN32Util::GetWindowsHDRStatus();
+  m_isHdrEnabled = (hdrStatus == HDR_STATUS::HDR_ON);
+  CLog::LogF(LOGINFO, "Storing Windows HDR switch state: {}",
+                 m_isHdrEnabled ? "ON" : "OFF");
   m_AutoSwitchHDR = CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
                         DX::Windowing()->SETTING_WINSYSTEM_IS_HDR_DISPLAY) &&
                     DX::Windowing()->IsHDRDisplay();
-
-  // Auto switch HDR only if supported and "Settings/Player/Use HDR display capabilities" = ON
-  if (m_AutoSwitchHDR)
-  {
-    bool streamIsHDR = (picture.color_primaries == AVCOL_PRI_BT2020) &&
```

Comment:
Not seems necessary remove this code block as anyway is need toggle HDR when not match with HDR content and "Match HDR on content" feature is ON.

Note that same occurs later in ProcessHDR() but is intended for switch in middle of stream, thinking in Live TV (channels switching or add-ons).

Reply:
Okay I understand. I just noticed in the log this code was creating situations where HDR would get toggled OFF and then back ON (too many switches) for HDR content